<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MImageBookReader</class>
 <widget class="QMainWindow" name="MImageBookReader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <property name="windowIcon">
   <iconset resource="resfile.qrc">
    <normaloff>:/new/icons/icons/image.png</normaloff>:/new/icons/icons/image.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="MDocHeader" name="wdgHeader" native="true"/>
    </item>
    <item>
     <widget class="QStackedWidget" name="stwImg">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="widget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QWidget" name="wdgImgZoomPanel" native="true">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>rotate</string>
                   </property>
                   <property name="buddy">
                    <cstring>cmbRot</cstring>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QComboBox" name="cmbRot">
                   <property name="sizeAdjustPolicy">
                    <enum>QComboBox::AdjustToContents</enum>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>zoom</string>
                   </property>
                   <property name="margin">
                    <number>0</number>
                   </property>
                   <property name="textInteractionFlags">
                    <set>Qt::NoTextInteraction</set>
                   </property>
                   <property name="buddy">
                    <cstring>sldZoom</cstring>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QToolButton" name="btZoomPlus">
                   <property name="toolTip">
                    <string>zoom +</string>
                   </property>
                   <property name="text">
                    <string>zoom +</string>
                   </property>
                   <property name="icon">
                    <iconset resource="resfile.qrc">
                     <normaloff>:/new/icons/icons/loupe_plus.png</normaloff>:/new/icons/icons/loupe_plus.png</iconset>
                   </property>
                   <property name="shortcut">
                    <string>Ctrl++</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QToolButton" name="btZoomMinus">
                   <property name="toolTip">
                    <string>zoom -</string>
                   </property>
                   <property name="text">
                    <string>zoom -</string>
                   </property>
                   <property name="icon">
                    <iconset resource="resfile.qrc">
                     <normaloff>:/new/icons/icons/loupe_minus.png</normaloff>:/new/icons/icons/loupe_minus.png</iconset>
                   </property>
                   <property name="shortcut">
                    <string>Ctrl+-</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="lblValue">
                   <property name="frameShape">
                    <enum>QFrame::Box</enum>
                   </property>
                   <property name="text">
                    <string notr="true">x1.00</string>
                   </property>
                   <property name="textFormat">
                    <enum>Qt::RichText</enum>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                   <property name="margin">
                    <number>3</number>
                   </property>
                   <property name="textInteractionFlags">
                    <set>Qt::NoTextInteraction</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QSlider" name="sldZoom">
                 <property name="minimum">
                  <number>10</number>
                 </property>
                 <property name="maximum">
                  <number>400</number>
                 </property>
                 <property name="value">
                  <number>100</number>
                 </property>
                 <property name="tracking">
                  <bool>true</bool>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QGridLayout" name="gridLayout">
                 <item row="0" column="0">
                  <widget class="QToolButton" name="btZ1">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="focusPolicy">
                    <enum>Qt::NoFocus</enum>
                   </property>
                   <property name="text">
                    <string notr="true">x1</string>
                   </property>
                   <property name="shortcut">
                    <string>Ctrl+1</string>
                   </property>
                   <property name="checkable">
                    <bool>false</bool>
                   </property>
                   <property name="checked">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QToolButton" name="btZ2">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="focusPolicy">
                    <enum>Qt::NoFocus</enum>
                   </property>
                   <property name="text">
                    <string notr="true">x2</string>
                   </property>
                   <property name="shortcut">
                    <string>Ctrl+2</string>
                   </property>
                   <property name="checkable">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <widget class="QToolButton" name="btZ3">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="focusPolicy">
                    <enum>Qt::NoFocus</enum>
                   </property>
                   <property name="text">
                    <string notr="true">x3</string>
                   </property>
                   <property name="shortcut">
                    <string>Ctrl+3</string>
                   </property>
                   <property name="checkable">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="3">
                  <widget class="QToolButton" name="btZ4">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="focusPolicy">
                    <enum>Qt::NoFocus</enum>
                   </property>
                   <property name="text">
                    <string notr="true">x4</string>
                   </property>
                   <property name="shortcut">
                    <string>Ctrl+4</string>
                   </property>
                   <property name="checkable">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QToolButton" name="btZD2">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="focusPolicy">
                    <enum>Qt::NoFocus</enum>
                   </property>
                   <property name="text">
                    <string notr="true">/2</string>
                   </property>
                   <property name="shortcut">
                    <string>Ctrl+Alt+2</string>
                   </property>
                   <property name="checkable">
                    <bool>false</bool>
                   </property>
                   <property name="checked">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QToolButton" name="btZD3">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="focusPolicy">
                    <enum>Qt::NoFocus</enum>
                   </property>
                   <property name="text">
                    <string notr="true">/3</string>
                   </property>
                   <property name="shortcut">
                    <string>Ctrl+Alt+3</string>
                   </property>
                   <property name="checkable">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="2">
                  <widget class="QToolButton" name="btZD4">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="focusPolicy">
                    <enum>Qt::NoFocus</enum>
                   </property>
                   <property name="text">
                    <string notr="true">/4</string>
                   </property>
                   <property name="shortcut">
                    <string>Ctrl+Alt+4</string>
                   </property>
                   <property name="checkable">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="3">
                  <widget class="QToolButton" name="btZD5">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="focusPolicy">
                    <enum>Qt::NoFocus</enum>
                   </property>
                   <property name="text">
                    <string notr="true">/5</string>
                   </property>
                   <property name="shortcut">
                    <string>Ctrl+Alt+5</string>
                   </property>
                   <property name="checkable">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTreeWidget" name="treeImages">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="contextMenuPolicy">
               <enum>Qt::DefaultContextMenu</enum>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="rootIsDecorated">
               <bool>false</bool>
              </property>
              <property name="itemsExpandable">
               <bool>false</bool>
              </property>
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
              <property name="expandsOnDoubleClick">
               <bool>false</bool>
              </property>
              <attribute name="headerVisible">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Image</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>filename</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="MImageWidget" name="iwImg" native="true"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="layGk"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="layCop"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>310</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>&amp;action</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;tools</string>
    </property>
    <addaction name="actionGk_Lat_dictionary"/>
    <addaction name="actionCoptic_dictionary"/>
    <addaction name="action_Hebrew_dictionary"/>
    <addaction name="actionNumeric_converter"/>
   </widget>
   <addaction name="menuAction"/>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="resfile.qrc">
     <normaloff>:/new/icons/icons/delete.png</normaloff>:/new/icons/icons/delete.png</iconset>
   </property>
   <property name="text">
    <string>&amp;close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionGk_Lat_dictionary">
   <property name="icon">
    <iconset resource="resfile.qrc">
     <normaloff>:/new/icons/icons/alfa2.png</normaloff>:/new/icons/icons/alfa2.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Gk/Lat dictionary</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+G</string>
   </property>
  </action>
  <action name="actionCoptic_dictionary">
   <property name="icon">
    <iconset resource="resfile.qrc">
     <normaloff>:/new/icons/icons/gima.png</normaloff>:/new/icons/icons/gima.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Coptic dictionary</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+C</string>
   </property>
  </action>
  <action name="actionNumeric_converter">
   <property name="icon">
    <iconset resource="resfile.qrc">
     <normaloff>:/new/icons/icons/numcnv.png</normaloff>:/new/icons/icons/numcnv.png</iconset>
   </property>
   <property name="text">
    <string>&amp;numeric converter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+N</string>
   </property>
  </action>
  <action name="action_Hebrew_dictionary">
   <property name="icon">
    <iconset resource="resfile.qrc">
     <normaloff>:/new/icons/icons/abgd.png</normaloff>:/new/icons/icons/abgd.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Hebrew dictionary</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+H</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="2" margin="2"/>
 <customwidgets>
  <customwidget>
   <class>MDocHeader</class>
   <extends>QWidget</extends>
   <header>mdocheader.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MImageWidget</class>
   <extends>QWidget</extends>
   <header>mimagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resfile.qrc"/>
 </resources>
 <connections/>
</ui>
